lib/src/image_dart_compiling/transformer.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:barback/barback.dart';
+import 'package:path/path.dart' show url;
+import 'dart:async';
+import 'package:scissors/src/utils/io_utils.dart' show readAll;
+import 'package:scissors/src/image_dart_compiling/image_dart_compiling.dart';
+import '../image_inlining/image_inliner.dart';
+
+const outputFileName = 'images.dart';
+
+/// Transformer that compiles several image files to a Dart source file with
+/// constant definitions that has these image files Base64-encoded.
+class DartImageCompiler extends AggregateTransformer
```

Comment:
Checking `Transformer` subclasses, I can see plenty of `Compiler`s ([exhibit 1](https://github.com/angular/angular/blob/master/modules_dart/transform/lib/src/transform/stylesheet_compiler/transformer.dart), [exhibit 2](https://github.com/angular/angular/blob/master/modules_dart/transform/lib/src/transform/template_compiler/transformer.dart)) but not a single `CompilingTransformer`. Can I leave it as it is?